moveit_ros/moveit_servo/tests/test_ros_integration.cpp

```diff
@@ -106,6 +106,43 @@ TEST_F(ServoRosFixture, testJointJog)
   moveit_servo::StatusCode status = status_;
   RCLCPP_INFO_STREAM(servo_test_node_->get_logger(), "Status after jointjog: " << static_cast<size_t>(status));
   ASSERT_EQ(status_, moveit_servo::StatusCode::NO_WARNING);
+
+  // Ensure deprecation warning is reported for displacements command
```

Comment:
Not a deprecation warning, right? Just a regular one?

Reply:
According to the documentation, displacement commands were previously available in MoveIt. So I would say it counts as deprecation. Or should I change the message to "not yet supported"?